pkg/agent/openflow/network_policy.go

```diff
@@ -773,9 +822,12 @@ func (c *client) applyConjunctiveMatchFlows(flowChanges []*conjMatchFlowContextC
 	return nil
 }
 
-// sendConjunctiveMatchFlows sends all the changed OpenFlow entries to the OVS bridge in a single Bundle.
-func (c *client) sendConjunctiveMatchFlows(changes []*conjMatchFlowContextChange) error {
+// sendConjunctiveFlows sends all the changed OpenFlow entries to the OVS bridge in a single Bundle.
+func (c *client) sendConjunctiveFlows(changes []*conjMatchFlowContextChange, actionFlows []binding.Flow) error {
 	var addFlows, modifyFlows, deleteFlows []binding.Flow
+	for _, actionFlow := range actionFlows {
+		addFlows = append(addFlows, actionFlow)
+	}
```

Comment:
ditto

Reply:
Sure. Will have this nit updated in the following PR.

Reply:
Updated